lib/reqIdGenFactory.js

```diff
@@ -21,6 +21,11 @@ function reqIdGenFactory (requestIdHeader, optGenReqId) {
   return genReqId
 }
 
+function getGenReqId (contextServer, req) {
+  const result = contextServer.genReqId.bind(contextServer, req)
```

Comment:
```suggestion
  return contextServer.genReqId.call(contextServer, req)
```

No need to create a binded function 👍

Reply:
Ended up not needing `call` method was able to do this
```
function getGenReqId (contextServer, req) {
  return contextServer.genReqId(req)
}
```